### PR TITLE
chore(package): update graphql to version 14.0.2

### PR DESCRIPTION
**Pull request by rmccar**
Closes #60

### What is the context of this PR?
update graphql to version 14.0.2

### How to review 
Make sure all tests pass and check in console.log when running through the system to check removal of deprecated values hasn't caused more issues

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   },
   "devDependencies": {
     "chalk": "^2.3.0",
-    "graphql": "^0.13.2",
+    "graphql": "^14.0.2",
```
Comment (tgandrews) [inline package.json]: We should also update `yarn.lock`

Comment (rmccar) [reply package.json]: Done!

**Comment by samiwel** [comment]
Seems like even adding the declaration at the top of the schema doesn't add the isDeprecated property to input type fields and field arguments.

To fix this we may need to change the build script as we discussed the other day.

**Comment by rmccar** [comment]
I think the git branches were getting a bit complicated so I have moved the upgrade to 14 onto its own PR. Then I will deal with the deprecation on another PR

**Pull request closed without merging**